### DISCUSSION TITLE
Extension activation failed: "No access to GitHub Copilot found.

### DISCUSSION THREAD

**Discussion by onlyjazz** [community/community#81758] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I'm unable to activate Github Copilot with a paid Copilot account.

**Steps to reproduce**
I'm currently logged in to Github as onlyjazz.
I have a paid Copilot account.

<img width="685" alt="Screenshot 2023-12-24 at 19 41 24" src="https://github.com/community/community/assets/333957/7f33faba-1141-462e-b508-277c114e8af0">

I installed Visual Studio Code and GitHub Copilot for VS Code v1.143.0

I get an error message:

_[INFO] [auth] [2023-12-24T17:34:56.032Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-12-24T17:34:56.033Z] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as riskgraph."_

I have another Github account called riskgraph - but I've refactored all the code into the onlyjazz account and I am not logged into this account on my Mac.

I believe that the bug was caused by the fact that I was logged into my GH riskgraph account the **very first time** I installed the VS Code plugin.

It seems like a config attribute was set somewhere on the Mac file system that points to riskgraph, this seems to cause the bug

I rm -rf ~/Library/Application Support/Code - and then reinstalled the Copilot extension on the assumption that maybe the riskgraph account was cached over there - but that didn't work

It would be great to have a solution 

Danny (@onlyjazz)

**Comment by ejmvar** [comment]
I'm facing this same issue under Linux and Windows.
In my case, My personal account *is already linked to my employer's organization*.
Tried logging in with my employer's account, asked for token, entered my private account token (because it's linked) and cannot get copilot activation (as github sent that message to my employer's email address).
How can this issue be solved?

**Comment by luklot1** [comment]
Hello, I have the same problem on macOS. Did you find a solution to the problem?

**Reply by rx-sgupta** [reply]
Hello, I have the same problem on macOS. Did you find a solution to the problem?
